datalake/src/main/java/com/sequenceiq/datalake/flow/dr/backup/DatalakeBackupActions.java

```diff
@@ -96,6 +104,17 @@ protected void doExecute(SdxContext context, DatalakeTriggerBackupEvent payload,
 
                 SdxCluster sdxCluster = sdxService.getById(payload.getResourceId());
                 eventSenderService.sendEventAndNotification(sdxCluster, context.getFlowTriggerUserCrn(), ResourceEvent.DATALAKE_BACKUP_IN_PROGRESS);
+                sdxStatusService.setStatusForDatalakeAndNotify(DatalakeStatusEnum.DATALAKE_BACKUP_INPROGRESS, ResourceEvent.DATALAKE_BACKUP_REQUESTED,
+                        "Datalake backup requested", sdxCluster.getId(), context.getFlowId());
+//                FlowLog flowLog = flowLogService.getLastFlowLog(context.getFlowId()).get();
```

Comment:
Any reason this is commented out and not just removed entirely?

Reply:
yes, I can be removed. I could have missed removing it.